components/Dashboard/Settings/DetailsForm.tsx

```diff
@@ -106,11 +131,28 @@ const DetailsForm: React.FC<DetailsFormProps> = ({ currentUser }) => {
                 <input
                   type="text"
                   name="handle"
-                  value={handle}
-                  className="j-field"
-                  onChange={(e) => setHandle(e.target.value)}
-                  ref={register({ required: t('profile.details.displayNameError') as string })}
-                  disabled={true}
+                  className={`j-field ${errors.handle ? 'is-invalid' : ''}`}
+                  defaultValue={currentUser.handle}
+                  ref={register({
+                    required: t('profile.details.displayNameError') as string,
```

Comment:
I'm guessing the return type of `t()` is `string | undefined` accounting for a missing translation, and thus the type assertion here. In this case I think something like `t('<string name>') || 'MISSING_STING'` or something that will show up noticeably in the UI would be preferable to the type assertion and possible throwing down the line.

Reply:
I changed this to the string interpolation. Just like it is in the `SignupForm`